app.js

```diff
@@ -30,28 +37,42 @@ const getProducts = async function () {
 };
 getProducts();
 
-// productsCenterDOM.innerHTML = fetch("products.json")
-//   .then((products) => products.json())
-//   .then((products) => {
-//     let results = "";
-//     for (let index = 0; index < products.length; index++) {
-//       results += `
-//      <article class="product">
-//         <div class="img-container">
-//             <img
-//                 src="${products[index].url}"
-//                 alt="product"
-//                 class="product-img"
-//             />
-//             <button class="bag-btn" data-id="1">
-//                 <i class="fas fa-shopping-cart"></i>
-//                 add to bag
-//             </button>
-//         </div>
-//         <h3>${products[index].name}</h3>
-//         <h4>$${products[index].price}</h4>
-//     </article>
-// `;
-//     }
-//     return results;
-//   });
+//Showing Cart
+cartDOM.addEventListener("click", showCart);
```

Comment:
Why do you need this code? It's not part of this ticket. Same goes for "Close Cart"

Reply:
showing and closing cart are of the same branch , if i remove one of them i will have to create  separate branch for that too.  Should i need to create  separate branch for each of them or it is fine this way ?

Reply:
i understood.